docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -284,22 +258,17 @@ export default function StyledComponentsDeep() {
 Material-UI has a rich theme structure that you can leverage for
 the color manipulations, the transitions, the media queries, and more.
 
+By using the Material-UI's Provider, the theme will be available for the
+context as well (emotion or styled-components, depending on your configuraiton).
```

Comment:
> By using the Material-UI's Provider, the theme will be available for the context of the styled engine too (emotion or styled-components, depending on your configuration).

Maybe this is more clear?

Reply:
Ah, right. I understand now. Yes, that's better. ("style engine" though, right?)

Also , either "the Material-UI theme provider", or "Material-UI's theme provider", rather than both "the" and "'s".

Reply:
Changed 👍